paddle/framework/grad_op_builder_test.cc

```diff
@@ -10,6 +10,11 @@ namespace framework {
 
 class NOP : public OperatorBase {
  public:
+  NOP(const std::string &type, const std::vector<std::string> &inputs,
```

Comment:
It seems strange that NOP does nothing but need so much stuff to initialize it...